service/controller/handler/values/create.go

```diff
@@ -9,34 +9,38 @@ import (
 	"github.com/giantswarm/microerror"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 
-	controllerkey "github.com/giantswarm/config-controller/service/controller/key"
+	"github.com/giantswarm/config-controller/service/controller/key"
 )
 
 func (h *Handler) EnsureCreated(ctx context.Context, obj interface{}) error {
-	app, err := controllerkey.ToAppCR(obj)
+	app, err := key.ToAppCR(obj)
 	if err != nil {
 		return microerror.Mask(err)
 	}
+	annotations := app.GetAnnotations()
+	if annotations == nil {
+		annotations = map[string]string{}
+	}
 
-	configVersion, ok := app.GetAnnotations()[annotation.ConfigVersion]
+	configVersion, ok := annotations[annotation.ConfigVersion]
 	if !ok {
-		h.logger.Debugf(ctx, "App CR %q is missing %q annotation", app.Name, annotation.ConfigVersion)
-		if _, ok := app.GetAnnotations()[PauseAnnotation]; ok {
-			err = h.removeAnnotation(ctx, &app, PauseAnnotation)
-			if err != nil {
-				return err
-			}
-		}
+		h.logger.Debugf(ctx, "App CR is missing %q annotation", annotation.ConfigVersion)
+		h.logger.Debugf(ctx, "cancelling handler")
+		return nil
+	}
+
+	if configVersion == "0.0.0" {
```

Comment:
It feels like `0.0.0` should be extracted as a named const in the `key` package.

Reply:
True. Let me know if the name sound ok to you.